index.html

```diff
@@ -0,0 +1,203 @@
+                    </div>
+                </div>
+            </div>
+                <div id="showMore" class="mobile"></div>
+                <button type="submit" class="btnLect mobile" onclick="toggleShow">
+                    MORE <img src="./Image/arrow_down.png" alt="">
+                </button> 
+            
+            <div class="row desktop">
+                <div class="lects">
+                    <img src="./Image/MketoN.jpg" alt="Dr M Mketo">
+                    <div class="flex-lects">
+                        <h4 class="name">Dr M Mketo</h4>
+                        <p class="title">Department of Chemistry</p>
+                        <p class="descr">Specialising in inorganic (heavy metals, non-metals and metalloids) and organic pollutants are a threats to living organisms</p>
+                    </div>
+                </div>
+                <div class="lects">
+                    <img src="./Image/Prof-Mantile-Lekala.jpg" alt="Prof M Lekala">
+                    <div class="flex-lects">
+                        <h4 class="name">Prof M Lekala</h4>
+                        <p class="title">Department of Physics</p>
+                        <p class="descr">Specialising in Theoretical Nuclear, Atomic and Molecular Physics and Computational Physics</p>
+                    </div>
+                </div>
+            </div>
+            <div class="row desktop">
+                <div class="lects">
+                    <img src="./Image/EevaRapoo.jpg" alt="Prof E Rapoo">
+                    <div class="flex-lects">
+                        <h4 class="name">Prof E Rapoo</h4>
+                        <p class="title">Chair of Statistics Department</p>
+                        <p class="descr">Specialising in Stochastics, Mathematics and Statistics Education and numerical analysis.</p>
+                    </div>
+                </div>
+                <div class="lects">
+                    <img src="./Image/Gaetan-Kabera.jpg" alt="Prof G Kabera">
+                    <div class="flex-lects">
+                        <h4 class="name">Prof G Kabera</h4>
+                        <p class="title">Statistics Lecturer</p>
+                        <p class="descr">Specialising in Optimal experimental Designs, Survival Analysis and Analytic Hierarchy Process.</p>
+                    </div>
+                </div>
+            </div>
+        </article>
+            
+        <article id="sponsor" class="partners1 desktop">
+            <div class="sponsors desktop">
+                <h3>Partners</h3>
+                <img src="./Image/title-line.png" alt="">
+            </div>   
+            <ul class="desktop">
+                <li>ALUMNI</li>
+                <li>Funza Lushaka</li>
+                <li>Department of Arts and Culture</li>
+                <li>NSFAS</li>
+            </ul>
+        </article>
+        <footer class="desktop">
+            <p>2015 Creative Commons South Africa. Some Rights Reserved.</p>
```

Comment:
- [ ] Great job with the footer :+1: I will only suggest a small style adjustment of this text which I have highlighted in the image below :100: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/235143611-62782ad8-5c8d-4f44-981e-4887d7c2a209.png)

- The Behance Design
![image](https://user-images.githubusercontent.com/102699958/235143697-0a25d73b-32bb-4ab9-9612-0c81235bdfe9.png)